common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -95,9 +95,18 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
   }
 
   _renderPagination(): ?React$Node {
-    if ((this.state.current_page === this.state.project_pages) || this.state.projects_loading) {
+    if (this.state.current_page === this.state.project_pages) {
```

Comment:
nit: When loading the last page of results, the button disappears

Reply:
It should render the Loading button even on the last page of results now, so long as this.state.projects_loading is true.